src/Tasks/Common/Resources/Strings.resx

```diff
@@ -859,4 +859,8 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1188: Package {0} {1} has a resource with the locale '{2}'. This locale is not recognized by .NET. Consider notifying the package author that it appears to be using an invalid locale.</value>
     <comment>Error code is NETSDK1188. 0 is a package name, 1 is a package version, and 2 is the incorrect locale string</comment>
   </data>
+  <data name="PReleaseRequiresEnvVarOnSln" xml:space="preserve">
+    <value>NETSDK1189: To use PackRelease or PublishRelease in solution projects, you must set the environment variable DOTNET_CLI_ENABLE_X_RELEASE_FOR_SOLUTIONS to true where X is PUBLISH or PACK. This will increase the time to publish a sln.</value>
```

Comment:
We should parameterize this method so it includes the actual names / values:

```suggestion
    <value>NETSDK1189: To use {0} in solution projects, you must set the environment variable {0} to true. This will increase the time to complete the operation.</value>
```

Reply:
Done, though I don't see why `complete the operation` is better because it requires more characters and also is less specific.